src/fgdpp/fgdpp.cpp

```diff
@@ -6,25 +6,27 @@
 #include <filesystem>
 #include <fstream>
 #include <regex>
+#include <iostream>
 
 using namespace fgdpp;
+using namespace sourcepp::detail;
 
 constexpr int FGDPP_MAX_STR_CHUNK_LENGTH = 1024;
 
-constexpr char singleTokens[] = "{}[](),:=+";
-constexpr FGD::TokenType valueTokens[] = {FGD::OPEN_BRACE, FGD::CLOSE_BRACE, FGD::OPEN_BRACKET, FGD::CLOSE_BRACKET, FGD::OPEN_PARENTHESIS, FGD::CLOSE_PARENTHESIS, FGD::COMMA, FGD::COLUMN, FGD::EQUALS, FGD::PLUS};
-constexpr enum ParseError tokenErrors[] = {ParseError::INVALID_OPEN_BRACE, ParseError::INVALID_CLOSE_BRACE, ParseError::INVALID_OPEN_BRACKET, ParseError::INVALID_CLOSE_BRACKET, ParseError::INVALID_OPEN_PARENTHESIS, ParseError::INVALID_CLOSE_PARENTHESIS, ParseError::INVALID_COMMA, ParseError::INVALID_COLUMN, ParseError::INVALID_EQUALS, ParseError::INVALID_PLUS};
+constexpr const char singleTokens[11] = R"({}[](),:=+)";
```

Comment:
Don't specify the length of the array please - it will break if someone modifies the string